tests/test_create_cadet_domains.py

```diff
@@ -23,30 +24,46 @@ def test_creating_domains_from_s3(self):
         domains.sort()
         assert domains == ["Courts", "HQ", "Prison", "Probation"]
 
+        # test for user mce
```

Comment:
Mat has a nice solution for sorting the events into categories to better check what events are produced. you should see it on rebase.